client/app/bundles/course/assessment/submission/containers/TestCaseView.jsx

```diff
@@ -163,8 +175,8 @@ class VisibleTestCaseView extends Component {
 
     return (
       <Card>
-        <CardHeader title={title} />
-        <CardText>
+        <CardHeader title={title} actAsExpander={expandable} showExpandableButton={expandable} style={headerStyle} />
+        <CardText expandable={expandable}>
```

Comment:
If I'm not wrong can do `<CardText expanadable>`?

Reply:
Actually i think that would be set to `expandable={true}`